packages/react/src/components/ColorPicker/ColorPicker.deprecated.test.tsx

```diff
@@ -1,56 +1,51 @@
+import '@testing-library/jest-dom';
 import * as React from 'react';
-import { mount, ReactWrapper } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 
 import { ColorPicker } from './ColorPicker';
-import { ColorPickerBase } from './ColorPicker.base';
-import { ColorSliderBase } from './ColorSlider/ColorSlider.base';
-import { setWarningCallback } from '../../Utilities';
-import type { IColorPickerState } from './ColorPicker.base';
-import type { IColorPickerProps, IColorPickerStrings } from './ColorPicker.types';
+import type { IColorPickerStrings } from './ColorPicker.types';
+import { setWarningCallback } from '@fluentui/utilities';
 
-describe('ColorPicker', () => {
-  let wrapper: ReactWrapper<IColorPickerProps, IColorPickerState, ColorPickerBase> | undefined;
+function getBySelector(container: HTMLElement, selector: string) {
+  return container.querySelector(selector);
+}
+function getByAllSelector(container: HTMLElement, selector: string) {
+  return container.querySelectorAll(selector);
+}
```

Comment:
Should you be importing these from `testUtilities` instead?

Reply:
yeah, at one point I move it there but obviously got overhwelmed by so many changes, so this PR is not "top notch" per say, which I think is feasible because the aim here is not to improve things rather than unblock us (the bigger picture)